src/api/stellarSdk.ts

```diff
@@ -0,0 +1,4 @@
+import * as StellarSDK from 'stellar-sdk';
+const server = new StellarSDK.Server('https://horizon-testnet.stellar.org');
+
+export { StellarSDK, server };
```

Comment:
Come to think of it... why is this file necessary at all?

Reply:
The purpose of this file is to keep the API call in one place so that the rest of the application can import it in a more centralized way. I understand that this is done when what is imported may change, and only one file needs to be changed. And that we will not stop using Stellar, obviously. If it's unnecessary, I'll remove it.

Reply:
Please remove